packages/common/src/ui/layout/tables/DataTable.tsx

```diff
@@ -116,16 +118,29 @@ export const DataTable = <T extends DomainObject>({
             );
           })}
         </TableBody>
+        <TableFooter
+          sx={{
+            flex: 1,
+            display: 'flex',
+            flexDirection: 'column',
+            position: 'sticky',
+            insetBlockEnd: 0,
+            backgroundColor: 'white',
+          }}
+        >
+          <TableRow>
+            {pagination && onChangePage && (
+              <PaginationRow
+                page={pagination.page}
+                offset={pagination.offset}
+                first={pagination.first}
+                total={pagination.total ?? 0}
+                onChange={onChangePage}
+              />
+            )}
+          </TableRow>
+        </TableFooter>
```

Comment:
Ah, also the pagination row was dropping to the bottom so this is trying to keep it sticky at the bottom